lib/tasks/multitenant/triggers.rake

```diff
@@ -38,6 +38,87 @@ namespace :multitenant do
     puts "Recreated #{triggers.size} triggers"
   end
 
+  desc 'Fix empty or corrupted tenant_id in accounts'
+  task :fix_corrupted_tenant_id_accounts, %i[batch_size sleep_time] => :environment do |_task, args|
+    batch_size = (args[:batch_size] || 100).to_i
+    sleep_time = (args[:sleep_time] || 1).to_i
+
+    ids = (Rails.application.simple_try_config_for(:corrupted_accounts) || [])
+
+    id_chunks = []
+    ids.each_slice(batch_size) { |slice| id_chunks << slice }
```

Comment:
You can use `in_groups_of(batch_size, false)`

https://apidock.com/rails/Array/in_groups_of